frontend/pages/login.tsx

```diff
@@ -1,125 +1,127 @@
-import type { NextPage } from "next";
-import * as React from "react";
-import { withLayout } from "@moxy/next-layout";
-import { LoginLayout } from "../layout/layout";
-
 import {
   Button,
   Card,
-  CardBody,
-  CardFooter,
-  CardHeader,
   Grid,
   Heading,
-  Link,
-  ListItem,
+  List,
   Stack,
   Text,
-  UnorderedList,
 } from "@chakra-ui/react";
-import { ExternalLinkIcon } from "@chakra-ui/icons";
 import { useLogin } from "../context/auth-context";
 import { Quotas } from "../components/quotas";
 import { TRANSCRIBE_QUOTAS } from "../model";
 import { formatDuration } from "date-fns";
+import { ExternalLink } from "@/components/externalLink";
+import { MappedIcon } from "@/components/mappedIcon";
+import { NextPageWithLayout } from "@/pages/_app";
+import { LoginLayout } from "../layout/layout";
 
-const Login: NextPage = () => {
-  const { handleLogin } = useLogin();
+const Login: NextPageWithLayout = () => {
+  const styles = {
+    root: {
+      bgColor: { base: "gray.100", _dark: "gray.700" },
+      borderColor: { base: "blue.900", _dark: "gray.800" },
```

Comment:
Use the default border colour

```suggestion
      bgColor: { base: "gray.100", _dark: "gray.700" },
```